app/Http/routes.php

```diff
@@ -26,6 +26,10 @@
     Route::model('competitions', 'Gladiator\Models\Competition');
     Route::resource('competitions', 'CompetitionsController');
 
+    // Competitions
+    Route::model('contest', 'Gladiator\Models\Contest');
```

Comment:
yeah ill change it, as I pointed out to Diego 'ContestsController' just sounded funny to me for some reason https://github.com/DoSomething/gladiator/pull/65#discussion_r55572885